build.cake

```diff
@@ -204,6 +208,17 @@ Task("Publish-Linux")
     DotNetCorePublish("./src/Ombi.Updater/Ombi.Updater.csproj", publishSettings);
 });
 
+Task("Publish-Linux-ARM")
+    .Does(() =>
+{
+    publishSettings.Runtime = "linux-arm";
+    publishSettings.OutputDirectory = Directory(buildDir) + Directory("netcoreapp2.0/linux-arm/published");
+
+    DotNetCorePublish("./src/Ombi/Ombi.csproj", publishSettings);
+    CopyFile(buildDir + "/netcoreapp2.0/linux-x64/Swagger.xml", buildDir + "/netcoreapp2.0/linux-arm/published/Swagger.xml");
```

Comment:
`/netcoreapp2.0/linux-x64/Swagger.xml` ? Shouldn't this be `/linux-arm/`?

Reply:
Will do.